liwords-ui/src/about/termsOfService.tsx

```diff
@@ -0,0 +1,444 @@
+import React, { useEffect } from 'react';
+import { TopBar } from '../topbar/topbar';
+import './staticContent.scss';
+import woogles from '../assets/woogles.png';
+import moment from 'moment';
+
+// Always change this when the terms change.
+const TERMS_VERSION_DATE = '2021-07-06';
+
+export const setTermsNotify = () => {
+  localStorage?.setItem('termsDate', Date.parse(TERMS_VERSION_DATE).toString());
+};
+
+export const getTermsNotify = () => {
+  const releaseDate = Date.parse(TERMS_VERSION_DATE);
+  const termsDate = parseInt(localStorage?.getItem('termsDate') || '0', 10);
+  return {
+    notified: releaseDate === termsDate,
+    changed: moment(TERMS_VERSION_DATE).format('MMMM Do, YYYY'),
```

Comment:
- US date format only
- just return `changed: TERMS_VERSION_DATE,` and let the caller format only when needed
- similarly the localStorage can store the TERMS_VERSION_DATE as-is (instead of epoch millis as string) to skip Date.parse/toString/parseInt

Reply:
US date format is entirely within local storage right? (in any case, this code has been backed out)